Dockerfile.build

```diff
@@ -1,13 +1,13 @@
 FROM golang:1.15-alpine AS builder
-ENV GO111MODULE=on
+ENV GO111MODULE on
 ENV XDG_CACHE_HOME="/go/src/github.com/mdblp/.cache"
 WORKDIR /go/src/github.com/mdblp/
 RUN apk --no-cache update && \
     apk --no-cache upgrade && \
-    apk add --no-cache gcc musl-dev git rsync python3 py3-pip make tzdata && \
+    apk add --no-cache gcc musl-dev git rsync python3 python3-dev py3-pip make tzdata && \

```

Comment:
what is the purpose of the `python3-dev` ?
is it to fix the build issue ?

Reply:
Yes for some reason, without python3-dev it do not work anymore
There is a gcc compile error: #include "python3.h" -> file not found